service/dap/server.go

```diff
@@ -1025,9 +1053,61 @@ func (s *Session) onLaunchRequest(request *dap.LaunchRequest) {
 	argsToLog.Cwd, _ = filepath.Abs(args.Cwd)
 	s.config.log.Debugf("launching binary '%s' with config: %s", debugbinary, prettyPrint(argsToLog))
 
+	var redirected = false
+	s.outputMode |= outputToStd
+	switch args.OutputMode {
```

Comment:
Just have two modes, one that sends stuff through DAP and one that writes to stdout/stderr.

Reply:
I hope there is a third mode that prints both.

Reply:
I don't see how it would be useful.

Reply:
This can be used to compare the difference between the output of the client and the server.
Sometimes when a program is debugged remotely, some components need to ensure that the output is printed to stdout/stderr.
As far as I know, there are programs that pass information to other services via stdout/stderr.
In this case, it is necessary to print both.

Reply:
This is only going to work if the remote instance of delve is started with the 'dap' command, which we do not recommend doing. Additionally it can create discrepancies because the program is not connected to the real pipe: you are not propagating a close from the debuggeee to delve's stdout/stderr and you aren't propagating the SIGPIPE in the other direction either.

Reply:
Still TBD.

Reply:
done.